packages/conventional-gitlab-releaser/src/cli.js

```diff
@@ -35,17 +35,43 @@ const cli = meow({
                                 This value is ignored if preset is specified
 
       -c, --context             A filepath of a javascript that is used to define template constntiables
-  `
-}, {
-  alias: {
-    u: 'url',
-    t: 'token',
-    p: 'preset',
-    k: 'pkg',
-    r: 'releaseCount',
-    v: 'verbose',
-    n: 'config',
-    c: 'context'
+  `,
+  flags: {
+    url: {
+      alias: 'u',
+      default: 'https://api.github.com',
```

Comment:
I think this should be the GitLab API end-point.

Reply:
Copy and paste messed up, sorry about that, updated now @hbetts